### ISSUE TITLE
The plot commands should only be executed once per load and once per parameter/bike change

### ISSUE THREAD

**Issue by moorepants** [moorepants/BicycleParameters#45]
If you add a print statement in cycle_app.py where plot_eigenvalues is called and refresh the app, you see that print statement displayed 3 times. For some reason that callback is executed too many times. If you change the bike selection it them seems to execute 4 times. These callbacks should only be executed once per change of any input box and ideally once per load.

@sandertyu

**Comment by sandertyu** [comment]
I agree this is an issue. It doesn't really seem to break anything at the moment but it certainly should be debugged. It's been happening ever since I originally implemented that callback, and I'm not at all sure why it's happening. I think we should take a look at it during our meeting this week.
